pie/storage/__init__.py

```diff
@@ -0,0 +1,80 @@
+from __future__ import annotations
+
+from nextcord.ext import commands
+
+from pie.storage.database import Storage
+
+
+def get_str(
+    module: commands.Cog, key: str, guild_id: int = 0, def_val: str = None
```

Comment:
I think the guild should be first, then `key`, then `default`.

Reply:
In that case there can't be default value for guild_id, since you can't mix keyword arguments with "normal" ones.

So, it can be made, but in that case we have to explicitly use guild_id, even if we want to store global value.

Reply:
That's fine by me, it will be more descriptive and explicit if you have to set the global value.